octobot/src/server/github_handler.rs

```diff
@@ -284,7 +284,16 @@ impl Handler for GithubHandler {
                     )
                     .await
                 {
-                    Ok(pr) => changed_pr = Some(pr),
+                    Ok(mut refetched_pr) => {
+                        if refetched_pr.draft != pull_request.draft {
+                            warn!(
+                                "Refetched pull request had mismatched draft: {:?} != {:?}",
```

Comment:
someone saw an issue where a PR was submitted for review, and had a jira reference, but the jira did not get tagged. this is a theory as to why that maybe happened. we'll see.

Reply:
there have been race conditions in the github api before.